tests/Doxyfile

```diff
@@ -673,7 +673,7 @@ INPUT_ENCODING         = UTF-8
 # *.hxx *.hpp *.h++ *.idl *.odl *.cs *.php *.php3 *.inc *.m *.mm *.dox *.py
 # *.f90 *.f *.for *.vhd *.vhdl
 
-FILE_PATTERNS          = *.cpp *.c *.h *.dox
+FILE_PATTERNS          = *.h *.dox
```

Comment:
The latter for now. Due to the heavy use of inlining I don't yet have a good sense how things will split across the .h and .c files.